<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Krivý &amp; Šustek - SIVPAVS 2015</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registrácia účastníka školenia</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titul:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>11</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Meno:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>11</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Priezvisko:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>11</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>11</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>11</y>
      <width>167</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>11</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>98</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefónne číslo:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>10</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Štát:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>11</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>11</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Obec:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>11</y>
      <width>179</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>11</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  PSČ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>11</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>701</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Vybrané školenia</string>
   </property>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>19</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>699</width>
       <height>199</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>41</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>10</y>
         <width>40</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Názov:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>10</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>10</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Pridať</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>57</x>
         <y>10</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>10</y>
         <width>54</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>  Termín:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>10</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>57</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>  Priorita:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>lineEdit_9</zorder>
   <zorder>lineEdit_10</zorder>
   <zorder>lineEdit_11</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButton</zorder>
   <zorder>frame_5</zorder>
   <zorder>frame_6</zorder>
   <zorder>scrollArea_2</zorder>
  </widget>
  <widget class="QFrame" name="frame_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Firma:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>11</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Pobočka:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>11</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  Pozícia:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>11</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_20">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>11</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>611</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Chybový výstup:</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>611</width>
      <height>121</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>609</width>
       <height>119</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Validovať</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>530</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zobraziť</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyčistiť</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
